global_footstep_planner/src/global_footstep_planner.cpp

```diff
@@ -281,7 +281,7 @@ void GlobalFootstepPlanner::updateDiscretePlan() {
 
   // publishDiscretePlan();
   // publishContinuousPlan();
-  // timer.report();
+  // timer.reportStatistics();
```

Comment:
Ok with leaving useful debugging messages commented, just wanted to make sure you want it here.

Reply:
Actually this entire package is pretty outdated, it was before local planner existed and was only there to add footsteps to a global plan for visualization/mpc. This seems like a decent PR to just remove it entirely.